core-bundle/src/Resources/contao/models/PageModel.php

```diff
@@ -859,6 +860,7 @@ public function loadDetails()
 		$this->groups = $this->protected ? StringUtil::deserialize($this->groups) : false;
 		$this->layout = $this->includeLayout ? $this->layout : false;
 		$this->cache = $this->includeCache ? $this->cache : false;
+		$this->alwaysLoadFromCache = $this->includeCache ? $this->alwaysLoadFromCache : $this->alwaysLoadFromCache;
```

Comment:
Shouldn't the else be `false`?

Reply:
Lol, of course! Good catch, fixed in 22e224d0a71e4aca278c6d76df3648f600746869.